src/main/java/frc/robot/RobotContainer.java

```diff
@@ -3,16 +3,21 @@
 import edu.wpi.first.wpilibj.XboxController;
 import edu.wpi.first.wpilibj2.command.Command;
 import edu.wpi.first.wpilibj2.command.button.JoystickButton;
-import frc.robot.subsystems.example.ExampleSubsystem;
+import edu.wpi.first.wpilibj2.command.button.Trigger;
+import frc.robot.subsystems.shooter.Hood;
+import frc.robot.subsystems.shooter.Shooter;
+import frc.robot.subsystems.shooter.commands.HoodDefaultCommand;
+import frc.robot.subsystems.shooter.commands.Shoot;
 import frc.robot.valuetuner.ValueTuner;
 import webapp.Webserver;
 
 public class RobotContainer {
-  // The robot's subsystems and commands are defined here...
-    public ExampleSubsystem exampleSubsystem = ExampleSubsystem.getInstance();
+    // The robot's subsystems and commands are defined here...
     private final XboxController xbox = new XboxController(Ports.Controls.XBOX);
     private final JoystickButton a = new JoystickButton(xbox, XboxController.Button.kA.value);
-
+    private final Trigger rt = new Trigger(() -> xbox.getRightTriggerAxis() > 0);
```

Comment:
"rt" implies this is an Xbox button, but it isn't.

Reply:
It uses the boolean supplier from the xbox controller.

Reply:
It does, but it returns a boolean, not the actual input from the button.

Reply:
Ah I see